src/singletons/Logging.cpp

```diff
@@ -25,7 +26,7 @@ void Logging::addMessage(const QString &channelName, MessagePtr message)
     auto it = this->loggingChannels_.find(channelName);
     if (it == this->loggingChannels_.end())
     {
-        auto channel = new LoggingChannel(channelName);
+        auto channel = new LoggingChannel(channelName, platformName);
```

Comment:
This would be an issue if two channels have the same name but are on different platforms, instead of there being a map with the channelName as the key pointing to a LoggingChannel, it should be a `Map<Platform, Map<ChannelName, LoggingChannel>>`